extensions/amp-embedly-card/amp-embedly-card.md

```diff
@@ -51,6 +51,10 @@ Within your AMP page, you can include one or multiple `amp-embedly-card` compone
 </amp-embedly-card>
 ```
 
+### Standalone use outside valid AMP documents
+
+Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP. You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Read more in our guide [Use AMP components in non-AMP pages](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/). To find the standalone version of `amp-embedly-card`, see [`bento-embedly-card`](./1.0/README.md).
```

Comment:
oops I never added this last sentence in the PRs I made, i'll add this at some point